internal/conf/config.yaml

```
@@ -192,4 +192,8 @@ output:
     password: secret      # mysql database user password
     database: birdnet     # mysql database name
     host: localhost       # mysql database host
-    port: 3306            # mysql database port
\ No newline at end of file
+    port: 3306            # mysql database port
+
+# Sentry telemetry configuration (opt-in, respects EU privacy laws)
+sentry:
+  enabled: false          # false by default, must be explicitly enabled by user (opt-in)
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)